Formula/nest.rb

```diff
@@ -0,0 +1,223 @@
+      # is compatible with brewed python2.7+
+      args << "-Dwith-python=ON"
+      python_exec = "python"
+    end
+
+    python_version = Language::Python.major_minor_version(python_exec)
+    bundle_path = libexec/"lib/python#{python_version}/site-packages"
+    bundle_path.mkpath
+
+    resource("nose").stage do
+      system python_exec, *Language::Python.setup_install_args(libexec)
+    end
+
+    resource("numpy").stage do
+      # see https://github.com/Homebrew/homebrew-core/blob/master/Formula/numpy.rb
+      system python_exec, "setup.py",
+        "build", "--fcompiler=gnu95", "--parallel=#{ENV.make_jobs}",
+        "install", "--prefix=#{libexec}",
+        "--single-version-externally-managed", "--record=installed.txt"
+    end
+
+    resource("scipy").stage do
+      # require numpy
+      ENV.prepend_create_path "PYTHONPATH", bundle_path
+      system python_exec, "setup.py",
+        "build", "--fcompiler=gnu95", "--parallel=#{ENV.make_jobs}",
+        "install", "--prefix=#{libexec}",
+        "--single-version-externally-managed", "--record=installed.txt"
+    end
+    (lib/"python#{python_version}/site-packages/homebrew-nest-dependencies.pth").write "#{bundle_path}\n"
+
+    resource("Cython").stage do
+      # is only a build dependency
+      system python_exec, *Language::Python.setup_install_args(buildpath/"cython")
+    end
+
+    # Add local build resource Cython residing in buildpath to paths
+    ENV.prepend_create_path "PATH", buildpath/"cython/bin"
+    ENV.prepend_create_path "PYTHONPATH", buildpath/"cython/lib/python#{python_version}/site-packages"
+
+    mkdir "build" do
+      system "cmake", "..", *args
+      system "make"
+      system "make", "install"
+    end
+
+    # Replace internally accessible gcc with externally accesible version
+    # in nest-config if required
+    inreplace bin/"nest-config",
+        %r{#{HOMEBREW_REPOSITORY}/Library/Homebrew/shims.*/super},
+        "#{HOMEBREW_PREFIX}/bin"
+  end
+
+  test do
+    # simple check whether NEST was compiled & linked
+    system bin/"nest", "--version"
+
+    # necessary for the python tests
+    ENV["exec_prefix"] = prefix
+    # if build.head? does not seem to work
+    if !File.directory?(pkgshare/"sources")
+      # Skip tests for correct copyright headers
+      ENV["NEST_SOURCE"] = "SKIP"
+    else
+      # necessary for one regression on the sources
+      ENV["NEST_SOURCE"] = pkgshare/"sources"
+    end
+
+    if build.with? "mpi"
+      # we need the command /mpirun defined for the mpi tests
+      # and since we are in the sandbox, we create it again
+      nestrc = <<-EOS
+        /mpirun
+        [/integertype /stringtype /stringtype]
+        [/numproc     /executable /scriptfile]
+        {
+         () [
+          (mpirun -np ) numproc cvs ( ) executable ( ) scriptfile
+         ] {join} Fold
+        } Function def
+      EOS
+      File.open(ENV["HOME"]+"/.nestrc", "w") { |file| file.write(nestrc) }
```

Comment:
`(buildpath/".nestrc").write nestrc`, no?

Reply:
nest will look in `HOME` for `.nestrc`... if `testpath` is equal to `HOME` in the testing-sandbox, then this would be ok. But since there will not be a MPI build, `.nestrc` is not needed and those lines will go anyway.

Reply:
Yeah, `testpath` is `HOME` here, as is `buildpath` during `def install`, although the latter may one day change as it has some side-effects that aren't completely desirable.